index.html

```diff
@@ -0,0 +1,162 @@
+                        computer
+                    </span>
+                    <h3 class="program-name">
+                        Hackathon Challenge
+                    </h3>
+                    <p class="program-description">
+                        The Hackathon will showcase Web3-based solutions for different sectors on the continent. The top three winners will be selected by judges.
+                    </p>
+                </div>
+                <div class="program-item">
+                    <span class="material-symbols-outlined">
+                        analytics
+                    </span>
+                    <h3 class="program-name">
+                        Breakout sessions
+                    </h3>
+                    <p class="program-description">
+                        We’ll get down to the nitty-gritty of topics such as decentralised finance (DeFI), (NFTs), and decentralised autonomous organizations (DAOs).
+                    </p>
+                </div>
+                <div class="program-item">
+                    <span class="material-symbols-outlined">
+                        museum
+                    </span>
+                    <h3 class="program-name">
+                        Exhibitions
+                    </h3>
+                    <p class="program-description">
+                        Get to interact with blockchain/Web3-focused brands and companies and discover how they provide their services.
+                    </p>
+                </div>
+                <div class="program-item">
+                    <span class="material-symbols-outlined">
+                        diversity_3
+                    </span>
+                    <h3 class="program-name">
+                        Networking 
+                    </h3>
+                    <p class="program-description">
+                        If you’re looking to immerse yourself deeply into any part of Web3, it will help if you get like-minded people going down the same road. 
+                    </p>
+                </div>
+            </div>
+            <a href="" class="program-btn"> Join Web3 Hackathon 2022 </a>
+        </div>
+    </section>
+    <section class="speaker-section" id="speaker">
+        <div class="featured-speakers">
+            <h2 class="speaker-title">
+                Featured Speakers
+            </h2>
+            <hr>
+            <div class="speaker-list">
+            </div>
+        </div>
+    </section>
+    <section class="partner-footer-mobile">
+        <h2 class="partner-title">
+            Partners
+        </h2>
+        <hr>
+        <div class="partner-logos">
+            <img src="./assets/images/FintechSurge-Logo_new.png" alt="FintechSurge Logo">
+            <img src="./assets/images/Gitex-logo-2.png" alt="Gitex logo">
+            <img src="./assets/images/NORTH-STAR-DUBAI-LOGO-New-2.png" alt="NORTH STAR DUBAI LOGO New">
+        </div>
```

Comment:
- [ ] In mobile view, these 2 sections are overlapping, kindly fix this issue 👍 

![image](https://user-images.githubusercontent.com/50721479/196672389-9b178cf4-1aee-4f86-8b2c-4df28736b738.png)